CHANGELOG.md

```diff
@@ -8,6 +8,13 @@ version number is tracked in the file `VERSION`.
 
 ## [Unreleased]
 ### Added
+
+### Changed
+
+### Fixed
+
+## [1.1.0] - 2022-03-31
+### Added
 - Added new `min_tls_version` features, which enables a `set_min_protocol_version` method on an `Ssl` object.
```

Comment:
The feature is actually called `early_access_min_tls_version` - shouldn't the changelog say this?